xesmf/tests/test_frontend.py

```diff
@@ -323,12 +342,22 @@ def test_regrid_dataset():
     # clean-up
     regridder.clean_weight_file()
 
+    # -----------------------------------------
     # testing locstream out
     regridder = xe.Regridder(ds_in, ds_locs, 'bilinear', locstream_out=True)
     ds_result = regridder(ds_in)
     # clean-up
     regridder.clean_weight_file()
 
+    # -----------------------------------------
+    # testing locstream in
+    regridder = xe.Regridder(ds_locs, ds_in, 'nearest_s2d', locstream_in=True)
+
+    outdata = regridder(ds_locs) 
+
+    # clean-up
+    regridder.clean_weight_file()
+
```

Comment:
(same as above) I would separate out of the tests for locstreams into different functions